solutions/devicesimulation/single-vm/setup.sh

```diff
@@ -23,22 +23,22 @@ export PCS_IOTHUB_CONNSTRING=""
 
 while [ "$#" -gt 0 ]; do
     case "$1" in
+        --solution-setup-url)      PCS_SOLUTION_SETUP_URL="$2" ;; # e.g. https://raw.githubusercontent.com/Azure/pcs-cli/DS-1.0.0/solutions/devicesimulation
         --subscription-domain)     PCS_SUBSCRIPTION_DOMAIN="$2" ;;
         --subscription-id)         PCS_SUBSCRIPTION_ID="$2" ;;
         --hostname)                HOST_NAME="$2" ;;
         --log-level)               PCS_LOG_LEVEL="$2" ;;
         --solution-type)           PCS_SOLUTION_TYPE="$2" ;;
         --solution-name)           PCS_SOLUTION_NAME="$2" ;;
         --resource-group)          PCS_RESOURCE_GROUP="$2" ;;
-        --iothub-name)             PCS_IOHUB_NAME="$2" ;;
-        --iothub-sku)              PCS_IOTHUB_SKU="$2" ;;
-        --iothub-tier)             PCS_IOTHUB_TIER="$2" ;;
-        --iothub-units)            PCS_IOTHUB_UNITS="$2" ;;
+        --iothub-name)             PCS_IOHUB_NAME="$2" ;;	
```

Comment:
remove empty spaces at the end of these lines

Reply:
fixed